.env

```diff
@@ -0,0 +1,9 @@
+MYSQLDB_USER=root
+MYSQLDB_ROOT_PASSWORD=root
+MYSQLDB_DATABASE=bookstore
+MYSQL_LOCAL_PORT=3307
+MYSQL_DOCKER_PORT=3306
+
+SPRING_LOCAL_PORT=8088
+SPRING_DOCKER_PORT=8080
+DEBUG_PORT=5005
```

Comment:
Don’t push `.env` file to GH (don’t forget remove it from the remote repository). It’s not secure. Add this file to `.gitignore` 
Instead of this file add `env.template` with variables without values.